### PR TITLE
added feature 2

### PR DESCRIPTION
**Pull request by Monishadhanasekar**
added new feature

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,2 @@
-console.log("This is popcorn")
\ No newline at end of file
+console.log("This is popcorn")
+console.log("This is feature 2")
```
Comment (Ajithkumarsekar) [inline index.js:2]: change this to `added feature 2`

**Review by Ajithkumarsekar — CHANGES_REQUESTED** [review]
Please correct the above mentioned changes

File: index.js (hunk 1 above)
Comment (Monishadhanasekar) [reply index.js:2]: okay

**Review by Ajithkumarsekar — APPROVED** [review]
LGTM

**Pull request merged**